randomkick.py

```diff
@@ -27,18 +33,39 @@
         while True:
             clicked.clear()
             users = await bot.get_participants(GROUP)
-            chosen = random.choice(users)
-            chosen.name = html.escape(utils.get_display_name(chosen))
             start = time.time()
+            if chosen is None or not (user for user in users if chosen.id == user.id):
+                bot_user = await bot.get_me()
+                chosen = bot_user
+                while chosen == bot_user or chosen.bot:
```

Comment:
This comparission will likely fail since comparing the entire user object is a bad idea. Comparing the ID would be enough. Checking if it's a `bot` is unnecessary (and you only do it here), and it's not related to this change.